### PR TITLE
OSDOCS#10302: Added the eu-central-2 region to the ROSA with HCP docs.

### PR DESCRIPTION
**Pull request by EricPonvelle**
Version(s):
`enterprise-4.15+`

Issue:
**[OSDOCS-10302](https://issues.redhat.com/browse/OSDOCS-10302)**

Link to docs preview:
- [ROSA with HCP regions](https://file.rdu.redhat.com/eponvell/OSDOCS-10302_eu-central-2-zone/rosa_architecture/rosa_policy_service_definition/rosa-hcp-service-definition.html#rosa-sdpolicy-regions-az_rosa-hcp-service-definition)
- Added a link to ROSA Classic and ROSA with HCP that points to the AWS regions: 
image
- [Release note](https://file.rdu.redhat.com/eponvell/OSDOCS-10302_eu-central-2-zone/rosa_release_notes/rosa-release-notes.html#rosa-q2-2024_rosa-whats-new)

QE review:
- [ ] QE has approved this change.

Additional information:
Added the `eu-central-2` region to ROSA with HCP docs.

### CODE REVIEW

**Comment by pawelpaszki** [comment]
looks good from osdfm QE perspective

File: modules/rosa-sdpolicy-am-regions-az.adoc [hunk 1]
```diff
@@ -60,6 +60,7 @@ endif::rosa-with-hcp[]
 ifndef::rosa-with-hcp[]
 * eu-west-3 (Paris)
 endif::rosa-with-hcp[]
+* eu-central-2 (Zurich)
```
Comment (jneczypor) [inline modules/rosa-sdpolicy-am-regions-az.adoc:63]: A couple of things to check:
1. It looks like you conditionalized this new region for ROSA with HCP, however, the `rosa list regions` command on the CLI shows that this region is also available for Classic. Should it be included there too?
2. eu-central-2 might be an AWS opt-in zone based on [this page from AWS](https://docs.aws.amazon.com/controltower/latest/userguide/opt-in-region-considerations.html). I'm not sure, but you might want to check it out.
3. Don't forget to add your new region (for HCP and Classic) to the ROSA "What's New" page.

**Review by jneczypor — COMMENTED** [review]
Hi! Just a couple of notes that should be quick fixes. Because I'm new to peer review, @michaelryanpeter will add his review after. Thanks for your patience!

File: modules/rosa-sdpolicy-am-regions-az.adoc (hunk 1 above)
Comment (EricPonvelle) [reply modules/rosa-sdpolicy-am-regions-az.adoc:63]: @pawelpaszki and @sdminonne - can you let me know if this region is available on Classic and if it's opt in? I think the former is true, but I do not know about the latter.

**Comment by EricPonvelle** [comment]
@michaelryanpeter and @jneczypor - I updated the PR. Can you double check that everything looks right?

**Pull request closed without merging**